### PR TITLE
UI rebased

### PR DESCRIPTION
**Pull request by danishyasin33**

### CODE REVIEW

File: ui/.eslintrc.json [hunk 1]
```diff
@@ -28,5 +28,5 @@
       "version": "detect"
     }
   },
-  "extends": ["@agoric", "plugin:react/recommended"]
```
Comment (danishyasin33) [inline ui/.eslintrc.json]: Why did you remove agoric lint? This means your code has not been linted properly

File: ui/package.json [hunk 2]
```diff
@@ -39,7 +40,7 @@
     "lint": "yarn lint:eslint",
     "lint:eslint": "eslint '**/*.{js,jsx}'",
     "lint-fix": "eslint --fix '**/*.{js,jsx}'",
-    "configure": "node use-on-chain-config",
```
Comment (danishyasin33) [inline ui/package.json]: Why?

File: ui/public/index.html [hunk 3]
```diff
@@ -25,7 +25,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
```
Comment (danishyasin33) [inline ui/public/index.html:28]: Should be Agoric AMM

File: ui/src/.prettierrc [hunk 4]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (danishyasin33) [inline ui/src/.prettierrc]: I don't think this file is needed

File: ui/src/App.js [hunk 5]
```diff
@@ -1,38 +1,59 @@
...
+import agoricLogo from "assets/agoric-logo.svg";
+import Swap from "components/Swap/Swap";
+import AssetWrapper from "context/AssetWrapper";
+import { Tab } from "@headlessui/react";
+import clsx from "clsx";
+import Liquidity from "components/Liquidity/Liquidity";
+import PoolWrapper from "context/PoolWrapper";
+import { motion } from "framer-motion";
+import WalletToast from "components/Wallet/WalletToast";
+import InformationToast from "components/components/InformationToast";
```
Comment (danishyasin33) [inline ui/src/App.js]: Add two spaces afterwards, would be done automatically after agoric linting is reinstated

File: ui/src/App.js [hunk 6]
```diff
@@ -1,38 +1,59 @@
...
-
+import React, { useState, useEffect } from "react";
+import { setToast } from "utils/helpers";
+import { useApplicationContext } from "context/Application";
+import "react-toastify/dist/ReactToastify.css";
+import "styles/globals.css";
+import agoricLogo from "assets/agoric-logo.svg";
+import Swap from "components/Swap/Swap";
+import AssetWrapper from "context/AssetWrapper";
+import { Tab } from "@headlessui/react";
```
Comment (danishyasin33) [inline ui/src/App.js]: What is headlessui?

File: ui/package.json [hunk 7]
```diff
@@ -3,16 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@agoric/assert": "*",
```
Comment (danishyasin33) [inline ui/package.json:6]: Did you define these values?

File: ui/src/App.js (hunk 6 above)
Comment (Hussain-7) [reply ui/src/App.js]: These are basically unstyled and fully accessible UI components, that are designed to integrate with Tailwind CSS

File: ui/public/index.html (hunk 3 above)
Comment (Hussain-7) [reply ui/public/index.html:28]: Was already done probably missed during merging

File: ui/package.json (hunk 2 above)
Comment (Hussain-7) [reply ui/package.json]: This setting allows us to use the default on-chain settings. This was added by default when I updated by agoric-sdk and ran agoric install.

File: ui/package.json (hunk 7 above)
Comment (Hussain-7) [reply ui/package.json:6]: was having issue with this version of agoric assert thats why I changed it but now it working fine.

Comment (danishyasin33) [reply ui/package.json:6]: Please revert to original versions and try to run it. Let's not change this unless required.

File: ui/src/App.js [hunk 8]
```diff
@@ -1,33 +1,55 @@
...
+          className="md:absolute pl-0 top-0 left-5 m-7"
         />
-        <Tab.Group defaultIndex={0} onChange={i => setIndex(i)}>
-          <Tab.List className="bg-white p-2 text-md shadow-red-light-sm rounded-sm mb-20 transition-all duration-300 ease-in-out">
-            <Tab className={''}>
+        <Tab.Group
+          defaultIndex={0}
+          onChange={i => {
+            setIndex(i);
+            console.log(i);
```
Comment (danishyasin33) [inline ui/src/App.js:48]: remove redundant console log

File: ui/src/components/Liquidity/Liquidity.jsx [hunk 9]
```diff
@@ -108,20 +126,44 @@ const Liquidity = () => {
...
+                >
+                  Remove
+                </div>
+              </div>
+              {tabIndex === 0 && (
+                <motion.div>
+                  {assetloader ? (
+                    <CustomLoader text="Loading Assets..." size={25} />
+                  ) : (
+                    <AddLiquidity />
```
Comment (danishyasin33) [inline ui/src/components/Liquidity/Liquidity.jsx:153]: Add liquidity should not show the central asset since all liquidity pairs are with the central asset and it does not make sense to have a liquidity pair with it. @Hussain-7 
There's a filter that filters out the central asset, something is broken there, please fix. It's somewhere in assetDialog or something, find the appropriate dialog and you shall find the filter somewhere there.
